### PR TITLE
2488 adds a release note for 0.6.0

### LINKED ISSUE #2488: Update changelog for v0.6.0 (8/July)
**Is your feature request related to a problem? Please describe.**
this is a sub-task of #2477

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

Fixes #2488

### Status
**Work in progress**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,10 +5,53 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.6.0] - 2021-07-08
+### Added
+* 10 new transforms, a masked loss wrapper, and a network adaptor for transfer learning
```
Comment (Nic-Ma) [inline CHANGELOG.md:10]: Could you please change to `adapter` as the class name is `NetAdapter`? https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/netadapter.py

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,53 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+* `unpack_items` mode for `apply_transform` and `Compose`
+* New event `INNER_ITERATION_STARTED` in the deepgrow interactive workflow
+* `set_data` API for cache-based datasets to dynamically update the dataset content
+* Fully compatible with PyTorch 1.9
+* `--disttests` and `--min` options for `runtests.sh`
+* Initial support of pre-merge tests with Nvidia Blossom system
+### Changed
+* Base Docker image upgraded to `nvcr.io/nvidia/pytorch:21.06-py3` from
+  `nvcr.io/nvidia/pytorch:21.04-py3`
+* Optionally depend on PyTorch-Ignite v0.4.5 instead of v0.4.4
```
Comment (Nic-Ma) [inline CHANGELOG.md:29]: But all the handlers, engines, etc. still optional import ignite v0.4.4. Maybe we can say "compatible with ignite >= 0.4.4"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,10 +5,53 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+* Unified the demo, tutorial, testing data to the project shared drive, and
+  [`Project-MONAI/MONAI-extra-test-data`](https://github.com/Project-MONAI/MONAI-extra-test-data)
+* Unified the terms: `post_transform` is renamed to `postprocessing`, `pre_transform` is renamed to `preprocessing`
+* Unified the postprocessing transforms and event handlers to accept the "channel-first" data format
```
Comment (Nic-Ma) [inline CHANGELOG.md:33]: `evenly_divisible_all_gather` and `string_list_all_gather` moved to `monai.utils.dist`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,10 +5,53 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+### Removed
+* Support of 'batched' input for postprocessing transforms and event handlers
+* `TorchVisionFullyConvModel`
+* `set_visible_devices`
```
Comment (Nic-Ma) [inline CHANGELOG.md:37]: Removed `SegmentationSaver` and `TransformsInverter` handlers, and `ToTensor` transform.

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,10 +5,53 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+### Fixed
+* Issue of handling big-endian image headers
+* Multi-thread issue for non-random transforms in the cache-based datasets
+* Persistent dataset issue when multiple processes sharing a non-exist cache location
+* Typing issue with Numpy 1.21.0
+* Loading model dictionary with `CheckpointLoader` when `strict_shape=False`
```
Comment (Nic-Ma) [inline CHANGELOG.md:43]: Issue of loading `model`, `optimizer`, etc. together with `CheckpointLoader` when `strict_shape=False`

**Review by Nic-Ma — COMMENTED** [review]
Hi @wyli ,

Thanks for the great summary, I put some minor comments inline.
Others look good to me.

Thanks.

File: CHANGELOG.md (hunk 2 above)
Comment (wyli) [reply CHANGELOG.md:29]: the support is actually dropped because we don't have tests for 0.4.4 anymore

**Review by Nic-Ma — APPROVED** [review]
thanks.

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,10 +5,56 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+  [`Project-MONAI/MONAI-extra-test-data`](https://github.com/Project-MONAI/MONAI-extra-test-data)
+* Unified the terms: `post_transform` is renamed to `postprocessing`, `pre_transform` is renamed to `preprocessing`
+* Unified the postprocessing transforms and event handlers to accept the "channel-first" data format
+* `evenly_divisible_all_gather` and `string_list_all_gather` moved to `monai.utils.dist`
+### Removed
+* Support of 'batched' input for postprocessing transforms and event handlers
+* `TorchVisionFullyConvModel`
+* `set_visible_devices` utility function
+* `SegmentationSaver` and `TransformsInverter` handlers 
+* `ToTensor` transform
```
Comment (Nic-Ma) [inline CHANGELOG.md:40]: Please remove this line according to our discussion.
Thanks.

**Pull request merged**